distributedlog-build-tools/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
```

Comment:
I like the idea to organize the checkstyle files into a separated module so that we don't need to have configuration on each module.

I think we should do the similar thing for findbugs settings. Can you move the findbug settings to builds-tool module? (It can be a separate jira and pull request).

Also it might be clear to move the eclipse format settings and the merge-pr script to builds-tool module too.